ground/src/main/java/com/google/android/ground/persistence/local/room/converter/ConverterExt.kt

```diff
@@ -112,7 +113,7 @@ fun JobEntityAndRelations.toModelObject(): Job {
     jobEntity.style?.toModelObject(),
     jobEntity.name,
     taskMap.toPersistentMap(),
-    jobEntity.suggestLoiTaskType?.let { Task.Type.valueOf(it) }
+    jobEntity.strategy.let { DataCollectionStrategy.valueOf(it) }
```

Comment:
Why not just `DataCollectionStrategy.valueOf(jobEntity.strategy)`?

Or. like other entities, I might expect to see `jobEntity.strategy.toModelObject()`

Reply:
> Why not just `DataCollectionStrategy.valueOf(jobEntity.strategy)`?

Because `strategy` might be null, and Enum.valueOf() doesn't accept nulls
> 
> Or. like other entities, I might expect to see `jobEntity.strategy.toModelObject()`

Because in `jobEntity` strategy is a String, and defining `fun String.?toModelObject()` would be too broad.